src/applications/check-in/pre-check-in/pages/Introduction/IntroductionDisplay.jsx

```diff
@@ -55,6 +59,24 @@ const IntroductionDisplay = props => {
       ),
     },
   ];
+  const isPhone = hasPhoneAppointments(appointments);
+
+  const handleStart = useCallback(
+    e => {
+      if (e?.key && e.key !== ' ') {
```

Comment:
Will this fire and record when any key is pressed?

Reply:
This early return with catch anything that isn't a spacebar or enter key press

Reply:
Oh right sorry I missed the return statement here. I'll blame it on looking at this late in the day on Friday. I'm used to checking for keys with e.keyCode or e.code and didn't realize e.key is undefined with the enter press which is interesting.